### PR TITLE
fix: make Android version deterministic

### PR DESCRIPTION
**Pull request by jaulz**
## Description

The Android version will be calculated based on the version string.

## Motivation and context

Currently, the Android version is not deterministic and always depends on the predecessor. This is a challenge if the Android version was accidentally overwritten in `app.json`.

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.
Please, please, please, don't send your pull request until all of the boxes are ticked.

- [X] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [X] My pull request addresses _exactly one_ patch/feature.
- [X] I have created a _branch_ for this patch/feature.
- [X] Each individual _commit_ in the pull request _is meaningful_.
- [X] I have _added tests_ to cover my changes.
- [X] If my change requires a change to the _documentation_, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

**Comment by byCedric** [comment]
Love the PR, thanks! I will look into this tonight (UTC+02:00)

**Comment by byCedric** [comment]
My bad, I changed codecov's behaviour. It won't spam anymore! Again, thanks a lot ❤️ Will take a detailled look tonight!

**Comment by jaulz** [comment]
No worries at all :) I'm just happy to see such quick reaction and openness for changes! Feel free to change the PR according to your needs.

File: src/version-bumpers/platform-android.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
+import * as semver from 'semver';
 import { getAndroidPlatform } from '../expo';
 import { VersionBumper } from '../types';
 
 const bumpPlatformAndroid: VersionBumper = (meta, context) => {
+	const sdkVersion = semver.coerce(meta.manifest.sdkVersion || 0);
 	const android = getAndroidPlatform(meta.manifest);
 	const oldVersion = android.versionCode || 0;
-	const newVersion = oldVersion + 1;
+    const nextRelease = semver.coerce(context.nextRelease!.version);
```
Comment (byCedric) [inline src/version-bumpers/platform-android.ts]: Could you indent this with a tab, so it fits the other indentation? 😄

Comment (byCedric) [reply src/version-bumpers/platform-android.ts]: Oh wait, I will do it, bit faster.

**Pull request merged**